### ISSUE TITLE
Error when decompressing a certain 7z file

### ISSUE THREAD

**Issue by startover205** [tsolomko/SWCompression#40]
Hello
I’m getting an LZMAError.notEnoughToRepeat error when uncompressing the 3rd sample file (sample-3.7z) downloaded from the website:
https://getsamplefiles.com/sample-archive-files/7z
*The error was thrown after a long wait.

The 7z file can be uncompressed correctly on my Mac using `The Unarchiver`.

The code I use:
```
let data = try Data(contentsOf: fileURL)
let entries = try SevenZipContainer.open(container: data)
```

Thank you!

**Comment by tsolomko** [comment]
Hi,

Thank you for reporting this issue. I had a quick a look at it, but unfortunately due to the lack of time at the moment I haven't managed to find its root cause yet.

With regards to this comment of yours:
> *The error was thrown after a long wait.

Please make sure that SWCompression is compiled in Release mode: it is much faster after that.

**Comment by tsolomko** [comment]
This issue was fixed in [4.8.5](https://github.com/tsolomko/SWCompression/releases/tag/4.8.5).
